### DISCUSSION TITLE
Groundwork Whitespace Mgmt

### DISCUSSION THREAD

**Discussion by JeremyDKellett** [USACE/groundwork#65] [General]
Anybody have any thoughts on dense flag on tables be more dense? Subjectively there seems to be a lot of whitespace even when dense is selected. FWIW I wasn't able to tell a difference between the table with dense or not.

**Comment by jbkolze** [comment]
@JeremyDKellett Right now the dense flag only decreases the vertical padding for cells from "4" to "2.5".  When you refer to whitespace, I assume you might be speaking more to the horizontal aspect of the table?  At the moment the table defaults to using the full width of its parent element.

We could have the dense flag also change the table width from min-w-full to w-fit, which would shrink the table horizontally to the minimum display size for the data.  You can also do that by passing a width className manually, but I could see the assumption for "dense" being that it would handle that part for you.  Ultimately there are a lot of ways to set it up, but ideally we'd use whatever route is most intuitive for the user.

**Reply by JeremyDKellett** [reply]
thanks, I'll look at the w-fit too.

When I saw the dense flag I thought it would be more dense then ~33% of the vertical padding (which my eye didn't notice so I've got to check that too).

**Comment by willbreitkreutz** [comment]
I'm not opposed to adding `denser` (or whatever) as an option too, if we feel like there's too much space left
